tests/atlas_basic_empty.tftest.hcl

```diff
@@ -15,7 +15,7 @@ run "atlas_basic_empty" {
   }
 
   assert {
-    condition     = mongodbatlas_project.project.name == "terraform-mongodbatlas-atlas-basic"
+    condition     = mongodbatlas_project.project[0].name == "terraform-mongodbatlas-atlas-basic"
```

Comment:
curious why this change was needed

Reply:
because now, if a project resource is created, it is created as a list of 1 element because of the condition:

`count  = var.use_existing_project ? 0 : 1`